### PR TITLE
GB11 site overview

### PR DESCRIPTION
**Pull request by kris-kolosek**

### CODE REVIEW

File: app/assets/stylesheets/pages/_site_overview.scss [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+.story-site-overview {
+  flex: 0 1 calc(30%);
+  padding: 24px 6vw 1vw;
+  width: 300px;
+
+  .story-default-img {
+    border-radius: 10px;
+  }
+
+  .form-title {
```
Comment (sniffler) [inline app/assets/stylesheets/pages/_site_overview.scss]: Rule set contains (11/10) properties

File: app/assets/stylesheets/pages/_site_overview.scss [hunk 2]
```diff
@@ -0,0 +1,50 @@
+.stories-site-overview {
+  background-color: $white;
+  display: flex;
+  flex-flow: row wrap;
+  font-family: -apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen,Ubuntu,Cantarell,Droid Sans,Helvetica Neue,sans-serif;
+  position: relative;
```
Comment (sniffler) [inline app/assets/stylesheets/pages/_site_overview.scss]: Properties should be ordered background-color, display, flex-flow, font-family, padding-left, padding-right, position

File: config/routes.rb [hunk 3]
```diff
@@ -7,7 +7,9 @@
   resource :profile
   resource :stories
   resource :story
+  resource :site_overview
```
Comment (salac-kolosek) [inline config/routes.rb]: Wait why is site_overview resource. Overview is page where you show stories.

Comment (kris-kolosek) [reply config/routes.rb]: Thanks for pointing it out!

**Pull request merged**